muxink/src/codec/bytesrepr.rs

```diff
@@ -0,0 +1,172 @@
+//! Bytesrepr encoding/decoding
+use std::{fmt::Debug, marker::PhantomData};
+
+use bytes::{Buf, Bytes, BytesMut};
+use casper_types::bytesrepr::{self, FromBytes, ToBytes};
+use thiserror::Error;
+
+use super::{DecodeResult, FrameDecoder, Transcoder};
+use crate::codec::DecodeResult::Failed;
+
+#[derive(Debug, Error)]
+pub enum Error {
+    #[error("bytesrepr error")]
+    BytesreprError(bytesrepr::Error),
+}
+
+/// A bytesrepr encoder.
+#[derive(Default)]
+pub struct BytesreprEncoder<T> {
```

Comment:
`Debug` here as well?

Reply:
Here you are as well :)
15a8bdef50fd39e62e3ca905437318e161f3689a